go.mod

```diff
@@ -0,0 +1,11 @@
+module go-rejson
+
+go 1.12
```

Comment:
```suggestion
```
Also, I think that hard binding 'go' directive value of '1.12' should be removed

xref: golang/go#30446

Reply:
> Also, I think that hard binding 'go' directive value of '1.12' should be removed
> 
> xref: [golang/go#30446](https://github.com/golang/go/issues/30446)

So this should be fine being omitted all together? @Shivam010

Reply:
Yes you dont need to specify the go directive. We should remove that. Reminds me that the builds are failing on older version. Maybe I should've retained the Gopkg.* files 🤔 . WDYT @Shivam010 how do we support pre-go 1.11.x versions ?

Reply:
> Yes you dont need to specify the go directive. We should remove that. Reminds me that the builds are failing on older version. Maybe I should've retained the Gopkg.* files 🤔 . WDYT @Shivam010 how do we support pre-go 1.11.x versions ?

I think omitting 1.12 directive and using module path `github.com/nitishm/go-rejson` will not fail builds for the older versions, as go semantics will help.